package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "description": "A Webpack loader that logs source to your console for debugging purposes.",
   "homepage": "https://github.com/ianwalter/debug-loader",
+  "repository": "ianwalter/debug-loader",
```

Comment:
Thanks. I'd rather it use the full syntax so you can tell where it's hosted by looking at it, e.g.:
```json
"repository" : {
  "type" : "git",
  "url" : "git@github.com:ianwalter/debug-loader.git"
}
```

Reply:
There you go. :)